database/src/main/java/com/firebase/ui/database/FirebaseArray.java

```diff
@@ -95,19 +96,27 @@ public void onChildMoved(DataSnapshot snapshot, String previousChildKey) {
     }
 
     public void onCancelled(DatabaseError firebaseError) {
-        // TODO: what do we do with this?
+        notifyCancelledListeners(databaseError);
```

Comment:
The symbol `databaseError` is not defined here, `firebaseError` is what you called the method argument.